xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -432,17 +432,19 @@
         <ReturnType>System.DateTime</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>The time that the certificate was verified expressed in local time.</summary>
+        <summary>Gets or sets the time for which the chain is to be validated, expressed in local time.</summary>
         <value>A <see cref="T:System.DateTime" /> object.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This property returns a value indicating the time the certificate was validated, expressed in local time. This is important when validating signed messages, since the signature must have been valid at the time of signing, not at the time of validation.  
-  
-   
-  
-## Examples  
+This property returns a value indicating the time the chain is to be validated, expressed in local time. This is important when validating signed messages, since the signature must have been valid at the time of signing, not at the time of validation.
```

Comment:
I don't know how much data to cram in here, but if we wanted to say more it would be something like

Every certificate in the chain must have a NotBefore time no later than this value and a NotAfter time no earlier than this value, or the `X509ChainStatusFlags.NotTimeValid` flag will be asserted.  This value may also be used when determining the revocation status of a certificate.